### PR TITLE
DFR-3334 Judge Approve Orders - approves or changes

### PR DESCRIPTION
**Pull request by so99y**
### Jira link

See [DFR-3334](https://tools.hmcts.net/jira/browse/DFR-3334)

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: definitions/contested/json/CaseEventToComplexTypes/DFR-3332-1-nonprod.json [hunk 1]
```diff
@@ -64,16 +84,27 @@
     "CaseEventID": "FR_approveOrders",
     "CaseFieldID": "judgeApproval1",
     "ListElementCode": "judgeDecision",
+    "EventElementLabel": "Is this ${judgeApproval1.titleInJudgeDecisionMessage} ready to be sealed and issued",
```
Comment (jthmcts) [inline definitions/contested/json/CaseEventToComplexTypes/DFR-3332-1-nonprod.json:87]: ? at the end?

File: definitions/contested/json/ComplexTypes/DFR-3332-nonprod.json [hunk 2]
```diff
@@ -82,6 +92,15 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
+  {
+    "LiveFrom": "29/10/2024",
+    "ID": "FR_judgeApproval",
+    "ListElementCode": "amendedDocument",
+    "FieldType": "Document",
+    "ElementLabel": "You must upload Microsoft Word documents. Document names should clearly reflect the party name, the type of hearing and the date of the hearing. For example “JonesFDA11Jul24”",
```
Comment (jthmcts) [inline definitions/contested/json/ComplexTypes/DFR-3332-nonprod.json:100]: Are we enforcing this file type restriction in the CCD config?

Comment (hamzahtahirhmcts) [reply definitions/contested/json/ComplexTypes/DFR-3332-nonprod.json:101]: [502b231](https://github.com/hmcts/finrem-ccd-definitions/pull/2138/commits/502b23132c1ca73e8db607f87988d4edda7d40df)

**Review by hamzahtahirhmcts — APPROVED** [review]

File: definitions/contested/json/CaseEvent/DFR-3332-nonprod.json
Comment (dawudgovuk) [inline definitions/contested/json/CaseEvent/DFR-3332-nonprod.json:1]: Add a new line at end of file

File: definitions/contested/json/CaseEventToComplexTypes/DFR-3332-1-nonprod.json [hunk 3]
```diff
@@ -64,16 +84,27 @@
...
     "LiveFrom": "29/10/2024",
     "ID": "FR_approveOrdersToJudgeApproval1",
     "CaseEventID": "FR_approveOrders",
     "CaseFieldID": "judgeApproval1",
     "ListElementCode": "isFinalOrder",
     "DisplayContext": "OPTIONAL",
-    "FieldShowCondition": "judgeApproval1.title=\"Draft Order\" AND judgeApproval1.judgeDecision=\"READY_TO_BE_SEALED\""
+    "FieldShowCondition": "judgeApproval1.docType=\"DRAFT_ORDER\" AND (judgeApproval1.judgeDecision=\"READY_TO_BE_SEALED\" OR judgeApproval1.judgeDecision=\"JUDGE_NEEDS_TO_MAKE_CHANGES\")"
   }
 ]
```
Comment (dawudgovuk) [inline definitions/contested/json/CaseEventToComplexTypes/DFR-3332-1-nonprod.json:110]: add a new line, repeat for the other files.

**Review by dawudgovuk — APPROVED** [review]

File: definitions/contested/json/CaseEventToComplexTypes/DFR-3332-1-nonprod.json (hunk 3 above)
Comment (hamzahtahirhmcts) [reply definitions/contested/json/CaseEventToComplexTypes/DFR-3332-1-nonprod.json:110]: Screenshot 2024-12-11 at 12 28 43

new line is already here

File: definitions/contested/json/CaseEvent/DFR-3332-nonprod.json
Comment (hamzahtahirhmcts) [reply definitions/contested/json/CaseEvent/DFR-3332-nonprod.json:1]: Screenshot 2024-12-11 at 12 29 24

New line is there

**Pull request merged**